### PR TITLE
"esm module interop flag" reflects  knex #3571

### PR DESCRIPTION
**Pull request by D10221**
Follows https://github.com/knex/knex/pull/3616
Assuming build step is after knex#merge

### CODE REVIEW

File: sections/transactions.js [hunk 1]
```diff
@@ -80,6 +80,75 @@ export default [
       });
     `
   },
+  {
```
Comment (kibertoad) [inline sections/transactions.js]: Was this part supposed to be included in this PR?..

Comment (D10221) [reply sections/transactions.js]: No, don't know who it got there :(

**Comment by D10221** [comment]
Did reset the branch to gh-pages
re-applied the changes
and forced update of this branch
If you are not happy I understand :) , I can do it again in a new branch .
Thanks

**Comment by kibertoad** [comment]
Thanks, looks good!

**Comment by D10221** [comment]
Thanks

**Pull request merged**